core/src/epicli/data/common/ansible/playbooks/roles/preflight/defaults/main.yml

```diff
@@ -1,14 +1,88 @@
----
-supported_oses:
-  - name: CentOS
-    version:
-      min: 7.6
-      max: 7.9
-  - name: RedHat
-    version:
-      min: 7.6
-      max: 7.9
-  - name: Ubuntu
-    version:
-      min: 18.04
-      max: 18.04
+---
+supported_oses:
+  - name: CentOS
+    version:
+      min: 7.6
+      max: 7.9
+  - name: RedHat
+    version:
+      min: 7.6
+      max: 7.9
+  - name: Ubuntu
+    version:
+      min: 18.04
+      max: 18.04
+
+unsupported_roles:
+  - distro: Ubuntu
+    arch: x86_64
+    roles: []  # all supported
+  - distro: Ubuntu
+    arch: aarch64
+    roles:
```

Comment:
On Ubuntu or Redhat we should not support any roles at this point

Reply:
ok